.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: elixir
-elixir: '1.7.3'
-otp_release: '21.3'
+elixir: '1.9.2'
+otp_release: '22.0.7'
```

Comment:
This and `.tool-versions` should likely be synchronized to use the same Erlang/OTP version.

Reply:
good catch, not sure how tool ver got bumped